buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
if we have some need in the future then handle it then. I honestly don't grok the work happening about adding a test matrix for javaee. Seems like an early experiment which is exactly the kind of things we could do better *outside* the production build servers via these templates. Didn't spot it being mentioned on mailing list or build.next - got any pointers ? 

The only deviation I see currently when setting up linkage is the list of dependencies that should be possible to encode as an argument to the template (or in worst case 2 templates)